specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-09-01/virtualMachineScaleSet.json

```diff
@@ -4937,6 +4974,10 @@
           "$ref": "#/definitions/AutomaticRepairsPolicy",
           "description": "Policy for automatic repairs."
         },
+        "resiliencyPolicy": {
```

Comment:
this needs to come at the end of VirtualMachineScaleSetProperties.properties. 
So after constrainedMaximumCapacity

Please do the same for adding it to VirtualMachineScaleSetUpdateProperties

Reply:
Addressed